lib/blocs/portfolio/open_orders/open_orders_cubit.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:bloc/bloc.dart';
+import 'package:dalal_street_client/config/get_it.dart';
+import 'package:dalal_street_client/config/log.dart';
+import 'package:dalal_street_client/constants/error_messages.dart';
+import 'package:dalal_street_client/grpc/client.dart';
+import 'package:dalal_street_client/proto_build/actions/CancelOrder.pb.dart';
+import 'package:dalal_street_client/proto_build/actions/GetMyOrders.pb.dart';
+import 'package:equatable/equatable.dart';
+
+part 'open_orders_state.dart';
+
+class OpenOrdersCubit extends Cubit<OpenOrdersState> {
+  OpenOrdersCubit() : super(OpenOrdersInitial());
+  Future<void> getOpenOrders() async {
+    try {
+      final GetMyOpenOrdersResponse resp = await actionClient.getMyOpenOrders(
+          GetMyOpenOrdersRequest(),
+          options: sessionOptions(getIt()));
+      if (resp.statusCode == GetMyOpenOrdersResponse_StatusCode.OK) {
+        emit(OpenordersSucess(resp));
+      } else {
+        emit(OpenorderFailure(resp.statusMessage));
+      }
+    } catch (e) {
+      logger.i(e.toString());
```

Comment:
```suggestion
      logger.e(e);
```
Log exceptions as errors everywhere